SPECS/dracut/dracut.spec

```diff
@@ -68,6 +70,13 @@ Requires:       %{name} = %{version}-%{release}
 %description tools
 This package contains tools to assemble the local initrd and host configuration.
 
+%package overlayfs
```

Comment:
For the megaraid subpackage, it adds a config file to `/etc/dracut.conf.d/` that enables the megaraid module in the initramfs. Should we do the same for `overlayfs`?

Or would it make sense to have 2 separate sub-packages:

- `overlayfs-module` that contains the code for the overlayfs module and
- `overlayfs` that adds the config file.

? (Or perhaps this is overkill.)

Reply:
You may consider adding a comment in `module-setup.sh`